config/vufind/contentsecuritypolicy.ini

```diff
@@ -0,0 +1,32 @@
+; Settings for Content Security Policy header; you can learn more here:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+[CSP]
+; It is possible to use CSP in report-only mode. See report_uri setting below.
+; This setting could be set differently for each APPLICATION_ENV value (written as
+; array key below). Please note, that Whoops error handler (enabled in development
+; mode) does not show correctly when report_only is false.
+report_only[production] = false
+report_only[development] = true
+
+; The nonce (number used once) - unique number for each request, it is strongly
+; recommend to keep this setting on
+use_nonce = true
+
+; Directives, you can find available directives, on this page:
+; https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+; For evaluation of CSP you can use this tool provided by Google:
+; https://csp-evaluator.withgoogle.com/
+[Directives]
+default-src[] = "'none'"
+script-src[] = "'strict-dynamic'"
+script-src[] = "'unsafe-inline'"
+script-src[] = "http:"
+script-src[] = "https:"
+connect-src[] = "'self'"
+style-src[] = "'self'"
+img-src[] = "'self'"
+font-src[] = "'self'"
+base-uri[] = "'self'"
+; Set URI which browser should use to report CSP violation; you should provide URL
+; when you enable report_only
+;report-uri[] = 'https://example.report-uri.com'
```

Comment:
The [documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) suggests that report-uri has been deprecated in favor of report-to, and that best practice is currently to populate both. Is the framework already accounting for this normalization, or do we need to make adjustments here?

Reply:
That is true, I do not know why I mixed it this way, I've fixed it